### PR TITLE
feat: add omnilock

### PR DESCRIPTION
**Pull request by Daryl-L**

### CODE REVIEW

**Comment by mkxbl** [comment]
need add omni-lock config to `getBridgeConfig api`，ref to https://github.com/nervosnetwork/force-bridge/pull/310/files#diff-9fb6b91bcd81055cb9b1aa701e137db4b50db57ec69c2edbc08f665d2552f23aL448-R455

File: offchain-modules/packages/x/src/ckb/tx-helper/generator.ts [hunk 1]
```diff
@@ -474,9 +481,62 @@ export class CkbTxGenerator extends CkbTxHelper {
...
+    }
+
     logger.debug(`txSkeleton: ${transactionSkeletonToJSON(txSkeleton)}`);
     logger.debug(`final fee: ${await this.calculateCapacityDiff(txSkeleton)}`);
-    return txSkeletonToRawTransactionToSign(txSkeleton);
+
+    return txSkeleton;
   }
 
   async lock(
```
Comment (mkxbl) [inline offchain-modules/packages/x/src/ckb/tx-helper/generator.ts:542]: 1. need remove pw-lock related code
2. need add omni-lock witness if senderLockscript === omniLockscript
3. need calculate signingEntry for user to sign

Comment (Daryl-L) [reply offchain-modules/packages/x/src/ckb/tx-helper/generator.ts:542]: This have already been modified and is under local-ci.

File: offchain-modules/packages/app-rpc-server/src/handler.ts [hunk 2]
```diff
@@ -198,7 +198,7 @@ export class ForceBridgeAPIV1Handler implements API.ForceBridgeAPIV1 {
     const burnTx = await ckbTxGenerator.burn(fromLockscript, payload.recipient, asset, BigInt(amount));
     return {
       network: 'Nervos',
-      rawTransaction: burnTx,
+      rawTransaction: transactionSkeletonToObject(burnTx),
```
Comment (mkxbl) [inline offchain-modules/packages/app-rpc-server/src/handler.ts:201]: `fn lock` need transactionSkeletonToObject too

**Pull request merged**